### ISSUE TITLE
Accessibility issue due to no alt text

### ISSUE THREAD

**Issue by terioteri** [websharks/wp-super-snow#8]
Great plugin! Because the flakes are decorative images we need a null empty alt text (alt="") added for accessibility so that screen readers can ignore the multiple flakes. Thank you!!

**Comment by raamdev** [comment]
@terioteri Thanks for the report! We'll get this fixed in the next release. :-)

**Comment by terioteri** [comment]
Thank you - much appreciated!
